src/android/SplashScreen.java

```diff
@@ -301,16 +305,9 @@ public void run() {
                 splashImageView.setMinimumWidth(display.getWidth());
 
                 // TODO: Use the background color of the webView's parent instead of using the preference.
-                splashImageView.setBackgroundColor(preferences.getInteger("backgroundColor", Color.BLACK));
+                splashImageView.setBackgroundColor(Color.parseColor(preferences.getString("SplashBackgroundColor", "#FFFFFF")));
```

Comment:
This should be in a separate PR. Preferably keep a single PR focus on a single change.

Also this changes the default behavior. The default was black. Why change to white.

Also the above todo mentioned to not use from preference. Changes seems to go against the previous desired suggestion.